### PR TITLE
Entity

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Entity(name = "reviews")
+public class Review {
+    @Id
+    private Long id;
+    private String summary;
+    @Column(length = 100000)
+    private String text;
+    private LocalDateTime reviewTime;
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private Product product;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java]: CascadeType.ALL: you will delete one Review and Product also will be deleted.

File: src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java]: same

**Review by dmytro-hryhoruk — COMMENTED** [review]
Let's not use CascadeType.All on @ManyToOne()https://stackoverflow.com/questions/13027214/what-is-the-meaning-of-the-cascadetype-all-for-a-manytoone-jpa-association

File: src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java (hunk 1 above)
Comment (maks45) [reply src/main/java/com/maks/durov/amazonreviewapp/entity/Review.java]: got it

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**